python_modules/libraries/dagster-aws/dagster_aws/ssm/parameters.py

```diff
@@ -7,16 +7,36 @@
 
 
 def construct_ssm_client(
-    max_attempts: int, region_name: Optional[str] = None, profile_name: Optional[str] = None
+    max_attempts: int,
+    region_name: Optional[str] = None,
+    profile_name: Optional[str] = None,
+    endpoint_url: Optional[str] = None,
+    use_ssl: bool = True,
+    aws_access_key_id: Optional[str] = None,
+    aws_secret_access_key: Optional[str] = None,
+    aws_session_token: Optional[str] = None,
+    verify=None,
```

Comment:
verify can be a boolean or a string according to boto3 documentation.

while verify=False for production use would not be quite advisable, I can envision that someone might want to point to a development endpoint with a self-signed certificate.

do you think it's worth supporting here?

Reply:
As all we are doing is passing it through to boto3, I think it is worth supporting. Certainly not recommended, but don't think it is worth blocking users on.